applications/FluidDynamicsApplication/tests/TwoFluidInletTest/parameters_mpi.json

```diff
@@ -0,0 +1,128 @@
+        "domain_size"     	: 3,
+        "solver_type"   	: "TwoFluids",
+        "model_import_settings"       	: {
+            "input_type"     : "mdpa",
+            "input_filename" : "TwoFluidInletTest/test_inlet"
+        },
+		"maximum_iterations"			: 3,
+    	"formulation": {
+        	"dynamic_tau": 1.0
+    	},
+    	"echo_level"					: 0,
+    	"time_order"					: 2,
+    	"compute_reactions"				: false,
+    	"reform_dofs_at_each_step"		: false,
+    	"relative_velocity_tolerance"	: 0.001,
+    	"absolute_velocity_tolerance"	: 0.00001,
+    	"relative_pressure_tolerance"	: 0.001,
+    	"absolute_pressure_tolerance"	: 0.00001,
+        "volume_model_part_name"     	: "Parts_Fluid",
+        "skin_parts"                 	: ["AutomaticInlet3D_Inlet","Outlet3D_Outlet","Slip3D"],
+        "no_skin_parts"              	: [],
+        "move_mesh_flag"				: false,
+        "time_stepping"  : {
+            "automatic_time_step" : false,
+            "time_step"           : 0.00002
+        },
+    	"linear_solver_settings": {
+            "solver_type" : "amgcl",
+            "smoother_type" :"damped_jacobi",
+            "krylov_type" : "lgmres",
+            "coarsening_type" : "aggregation",
+            "max_iteration" : 500,
+            "provide_coordinates" : false,
+            "gmres_krylov_space_dimension" : 500,
+            "verbosity" : 0,
+            "tolerance" : 1e-8,
+            "scaling" : false,
+            "block_size" : 3,
+            "use_block_matrices_if_possible" : true,
+            "coarse_enough" : 500
+    	}
+    },
+	"processes"	:	{
+    	"initial_conditions_process_list"  : [],
+    	"boundary_conditions_process_list" : [{
+        "python_module" : "apply_two_fluids_inlet_process",
+        "kratos_module" : "KratosMultiphysics.FluidDynamicsApplication",
+        "Parameters"    : {
+            "model_part_name" 			: "AutomaticInlet3D_Inlet",
+            "interface_settings": {
+                "interface_normal": [0.0,0.0,1.0],
+                "point_on_interface": [0.0,0.0,0.5],
+                "inlet_transition_radius": 0.5
+            },
+            "fluid_1_settings" : {
+                "model_part_name": "fluid_1_inlet",
+                "variable_name": "VELOCITY",
+                "modulus": "2.0*z",
+                "constrained": true,
+                "direction": [1.0,0.0,0.0],
+                "interval": [0.0,"End"]
+            },
+            "fluid_2_settings" : {
+                "model_part_name": "fluid_2_inlet",
+                "variable_name": "VELOCITY",
+                "modulus": 0.1,
+                "constrained": true,
+                "direction": [1.0,0.0,0.0],
+                "interval": [0.0,"End"]
+            }
+        }
+    },{
+        "python_module" : "apply_outlet_process",
+        "kratos_module" : "KratosMultiphysics.FluidDynamicsApplication",
+        "Parameters"    : {
+            "model_part_name"    : "Outlet3D_Outlet",
```

Comment:
ideally you would prepend the MainModelpart-Name, but this can be done in a future PR

Eventually not prepending the MainModelPart will no longer be supported, and then we have to update all the tests.

The GiD-interface should implement this for a while already, maybe you can update

Reply:
Thank you for the hint. Do I understand correctly, that with this prefix it would be something like `MainModelPart.Outlet3D_Outlet`? Is that meant by "prepending the MainModelPart"?

Reply:
in your case i would be `FluidModelPart.Outlet3D_Outlet`